### ISSUE TITLE
Taiwan nicename is incorrect

### ISSUE THREAD

**Issue by smallma** [beyondplus/flags-dropdown-vue#3]
In your vue component, Taiwan nicename is Taiwan, Province of China. This is incorrect nicename. You should rename to "Taiwan, Republic of China". Please refer to the below.

https://en.wikipedia.org/wiki/Taiwan

**Comment by andrewintw** [comment]
No, Taiwan actually is not part of China. Does China really govern Taiwan? Do Taiwan people pay any tax to China? Does Taiwan use the same constitution with China? Does Taiwan use the same currency-codes with China? All the answer is NO. China people always like to deceive themselves.
